build.gradle

```diff
@@ -73,6 +77,8 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.2.1'
         classpath 'com.google.gms:google-services:4.1.0'
+        //noinspection GradleDependency
```

Comment:
this noinspection line should be removed, it is outdated and no longer applicable